### PR TITLE
[DO NOT MERGE] added support for foreman-proxy pulp plugin to installer config and answers

### PR DESCRIPTION
**Pull request by dmitri-d**
Pls. do not merge before https://github.com/theforeman/puppet-foreman_proxy/pull/107 has been merged.

### CODE REVIEW

**Comment by dmitri-d** [comment]
@domcleal: ping?

**Comment by domcleal** [comment]
I saw the first e-mail :)

File: config/answers.yaml [hunk 1]
```diff
@@ -32,3 +32,5 @@ foreman::compute::openstack: false
 foreman::compute::ovirt: false
 foreman::compute::rackspace: false
 foreman::compute::vmware: false
+foreman_proxy::plugin::pulp: false
+
```
Comment (ekohl) [inline config/answers.yaml]: Not a fan of empty lines at the end.

File: config/foreman-installer.yaml [hunk 2]
```diff
@@ -88,3 +88,8 @@
   :foreman::compute::ovirt:
     :dir_name: foreman
     :manifest_name: compute/ovirt
+  :foreman_proxy::plugin::pulp: 
+    :manifest_name: plugin/pulp
+    :params_name: plugin/pulp/params
+    :dir_name: foreman_proxy
+
```
Comment (ekohl) [inline config/foreman-installer.yaml]: Not a fan of empty lines at the end.

**Comment by dmitri-d** [comment]
@ekohl: fixed.

**Comment by domcleal** [comment]
Merged as 01186b1, thanks!

**Pull request closed without merging**